src/snapping/util.js

```diff
@@ -48,5 +53,40 @@ exports.getBufferLayer = (bufferLayerId, rootLayer, snapDistance) => {
   return bufferLayer;
 };
 
-exports.findVertexInCircle = (feature, circle) =>
+const findVertexInCircle = (feature, circle) =>
   getCoords(feature).find((coord) => pointInPolygon(coord, circle));
+
+exports.findVertexInCircleMulti = (snapGeom, circle, hoverPoint) => {
+  let vertex;
+  if (Array.isArray(snapGeom.geometry.coordinates[0][0])) {
```

Comment:
This would be nice as a descriptive variable, do I have it right that this is checking whether the feature is a multi-geometry feature? Or, is this specifically a multi-line feature at this point? Would be good to document it and a well-named variable would be a good way:

```suggestion
  const isMultiGeom = Array.isArray(snapGeom.geometry.coordinates[0][0]);
  if (isMultiGeom) {
```